packages/ui/react/src/components/Forms/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,185 @@
+import '@testing-library/jest-dom'
+import { render, screen, fireEvent } from '@testing-library/react'
+import { ThemeProvider } from 'styled-components'
+import { describe, expect, it, vi } from 'vitest'
+
+import { Alert } from './Alert'
+
+const theme = {}
+const renderWithTheme = (ui: React.ReactElement) =>
+  render(<ThemeProvider theme={theme}>{ui}</ThemeProvider>)
+
+vi.mock('src/colors', () => ({
+  getColor: (color: string) =>
+    ({
+      'blue.950': 'rgb(23, 37, 84)',
+      'red.500': 'rgb(239, 68, 68)',
+      'red.300': 'rgb(248, 113, 113)',
+      'red.200': 'rgb(254, 202, 202)',
+      'amber.400': 'rgb(251, 191, 36)',
+      'amber.200': 'rgb(254, 215, 170)',
+      'amber.100': 'rgb(254, 243, 199)',
+      'green.600': 'rgb(22, 163, 74)',
+      'green.300': 'rgb(134, 239, 172)',
+      'green.150': 'rgb(187, 247, 208)',
+    })[color],
+}))
+
+vi.mock('src/icons', () => ({
+  iconLoader: (name: string, size: number) => (
+    <svg
+      data-testid={`icon-${name}`}
+      width={size}
+      height={size}
+    />
+  ),
+}))
+
+describe('Alert', () => {
+  describe('renders different alert types', () => {
+    it('renders error alert with correct icon and text', () => {
+      renderWithTheme(
+        <Alert
+          type="error"
+          alertDescription="Error occurred"
+        />
+      )
+      expect(screen.getByText('Error occurred')).toBeInTheDocument()
+      expect(screen.getByTestId('icon-error')).toBeInTheDocument()
+    })
+
+    it('renders success alert with correct icon and text', () => {
+      renderWithTheme(
+        <Alert
+          type="success"
+          alertDescription="Success!"
+        />
+      )
+      expect(screen.getByText('Success!')).toBeInTheDocument()
+      expect(screen.getByTestId('icon-checkCircle')).toBeInTheDocument()
```

Comment:
@unnati06 the success icon was renamed, can you update the test accordingly?

https://github.com/devopness/devopness/blob/ebc1d03259e725903adc0590865af3cb2f66a73f/packages/ui/react/src/icons/iconLoader.tsx#L185

```suggestion
      expect(screen.getByTestId('icon-success')).toBeInTheDocument()
```